tests/js-tests/src/SceneTest/SceneTest.js

```diff
@@ -103,7 +103,8 @@ var SceneTestLayer1 = cc.Layer.extend({
 
         director.pushScene(new cc.TransitionSlideInT(1, scene));
     },
-    onQuit:function (sender) {
+    onExit:function (sender) {
+        cc.director.setNotificationNode(null);
```

Comment:
This line is fix another bug in comment: "Fix click setNotificationNode button in SceneTest then return to main menu the animation is still play on screen"